src/main/java/frc/robot/RobotContainer.java

```diff
@@ -90,6 +99,13 @@ public RobotContainer(RobotRunType runtimeType) {
      * {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
      */
     private void configureButtonBindings(RobotRunType runtimeType) {
+        algaeInIntake.whileTrue(leds.blinkLEDs(LEDPattern.solid(Color.kCyan)));
```

Comment:
Why pass a pattern vs keeping the pattern in the command and just pass a color??

Reply:
Also this will only start when the current spikes which stops the motors which will then drop the current and this will end almost instantly. Probably want to do an on true and use a timeout to blink a consistent amount of time each time